health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -152,16 +152,15 @@ const Hypothesis = ({ category, assumptions: initialAssumptions, setShowToast, a
       <Card>
         {assumptions.map((item, index) => {
           return (
-            <LabelFieldPair className="mp-hypothesis-label-field" style={{ alignItems: "center"}} key={index}>
+            <LabelFieldPair className="mp-hypothesis-label-field" key={index}>
               <div className="assumption-label">
                 <span>
                   {`${t(item)}`}
-                  <span className="mandatory-span">*</span>
+                  <span className="icon-wrapper">
+                    <TooltipWrapper content={t(`HYPOTHESIS_MESSAGE_FOR_${item}`)} children={<InfoBannerIcon fill={"#C84C0E"} />} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)